smart_open/gcs.py

```diff
@@ -561,7 +570,7 @@ def _upload_final_part(self):
           "%i bytes (total %.3fGB)"
           "headers %r",
           part_num,
-          part_size,
+          content_length,
```

Comment:
The formatstring can fit on one line.

Please use indendation that is a multiple of 4, here and everywhere.